src/features/home/components/modules/HomeModule.native.test.tsx

```diff
@@ -71,11 +108,90 @@ describe('<HomeModule />', () => {
       expect(screen.queryByText(highlightOfferModuleFixture.highlightTitle)).toBeNull()
     })
   })
+
+  it('should display BusinessModule', async () => {
+    renderHomeModule(formattedBusinessModule)
+
+    expect(await screen.findByText('Débloque ton crédit\u00a0! ')).toBeTruthy()
+  })
+
+  it('should display CategoryListModule', async () => {
+    renderHomeModule(formattedCategoryListModule)
+
+    expect(await screen.findByText('Cette semaine sur le pass')).toBeTruthy()
+  })
+
+  it('should display RecommendationModule', async () => {
+    const recommendedOffers = {
```

Comment:
suggestion (non-blocking): je recommande d'ajouter explicitement le typage (provenant de `api`) car si l'API change, il faudra que ce test change

Reply:
je change `recommendedOffers` par `recommendedOffersApi`

Reply:
le nommage est ok

j'aimerais avoir un type explicite

```suggestion
    const recommendedOffers: RecommendedOffers = {
```

(je ne sais pas si `RecommendedOffers` est le bon nom)

Reply:
Ah ! j'avais mal compris, dans ce cas je garde recommendedOffers en nommage et j'ajoute `SimilarOffersResponse` en typage !